chef_master/source/supermarket_backup_restore.rst

```diff
@@ -3,21 +3,96 @@ Supermarket Backup and Restore
 =====================================================
 `[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/supermarket_backup_restore.rst>`__
 
-Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration and to help ensure that important data can be restored, if required.
+Periodic backups of Supermarket data are an essential part of managing and maintaining a healthy configuration, and to help ensure that important data can be restored if required. In a typical installation of Supermarket, both the cookbook store and the database need to be backed up on a regular basis.
 
-**Cookbooks**
 
-The default location of cookbooks is here: `/var/opt/supermarket/data`
+Backup
+=====================================================
+
+Cookbook Backup
+-----------------------------------------------------
+
+If Supermarket is not configured to use AWS S3 storage for cookbooks, then the local cookbook storage location on the Supermarket server will also need to be backed up. 
```

Comment:
@robbkidd How is this?

Reply:
I think this is an English thing that could be tripping me up (and that I could be wrong about) on the "also." At the point the reader has arrived at this page, they haven't been told to back up anything yet. But the "also" makes it seem like we're adding an item to a list that only just started. So I think this can read "... will need to be backed up."

Reply:
Fixed!